### DISCUSSION TITLE
Fake Cash app Running. How to report abuse it?

### DISCUSSION THREAD

**Discussion by hula222** [community/community#112602] [Discussions] [answered]
### Select Topic Area

General

### Body

Is it easy to run Fake Website. There is fake Cash app customer service number link is running on google who is scamming people very easily. and guess what Github report abuse link is very hard to report such link./

Link is : https://cash-team-upp.github.io/support-team/

https://cash-team-upp.github.io/support-team/ 
Fake cash app customer service .

No one to report it

**Comment by luckymanbet** [comment]
If you've encountered a fake Cash App running and want to report abuse or fraudulent activity, here are the steps you can take:

1. Contact Cash App Support: The first and most direct step is to reach out to Cash App's support team. You can report the issue through the app itself or visit their website for contact information.

2. Provide Details: When reporting the fake Cash App, provide as much detail as possible about the fraudulent activity you've encountered. Include any relevant screenshots, transaction details, or other evidence that can help support your claim.

3. File a Complaint with Authorities: If you believe you've been a victim of a scam or fraud, you should also consider filing a complaint with relevant authorities, such as the Federal Trade Commission (FTC) in the United States or similar consumer protection agencies in your country.

4. Warn Others: Spread awareness about the fake Cash App to prevent others from falling victim to the same scam. You can do this by sharing your experience on social media platforms or forums frequented by Cash App users.

5. Stay Vigilant: Moving forward, exercise caution when using financial apps like Cash App. Be wary of any suspicious activity, requests for personal information, or unexpected transactions.

6. Secure Your Account: If you suspect that your own Cash App account has been compromised, take immediate steps to secure it. Change your password, enable two-factor authentication if available, and review your transaction history for any unauthorized activity.

By taking these steps, you can help combat fraudulent activity and protect yourself and others from falling victim to fake Cash App schemes.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! 

Thanks for the feedback - if you're unable to report using our tools, the best place to go for in situations like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
